eng/tools/TypeSpecValidation/src/rules/file-structure.ts

```diff
@@ -0,0 +1,28 @@
+import { Rule } from "../rule.js";
+import { RuleResult } from "../rule-result.js";
+import { runCmd } from "../utils.js";
+
+export class FileStructureRule implements Rule {
+  readonly name = "FileStructure";
+  readonly description = "Verify spec directory's file structure and naming conventions.";
+  async execute(folder: string): Promise<RuleResult> {
+    let success = true;
+    let errorOutput = "";
+
+    const tspConfig = (
+      await runCmd(`(Get-ChildItem -path ${folder} tspconfig.* -Recurse).Name`, process.cwd())
```

Comment:
Replace with package `globby`.  Already used in `typespec` repo:

https://github.com/microsoft/typespec/blob/ee3c4fa2f615293747739a739b05353b3d28073f/packages/compiler/package.json#L79